### PR TITLE
Feat/merchant items show page

### PR DESCRIPTION
**Pull request by devAndrewK**
Added tests establishing existence of correct attributes and only for that item. Added show page with attribute details and an update item link to be implemented later. Added model method to convert cents to dollars to be formatted in the view.

### CODE REVIEW

File: app/controllers/merchant_items_controller.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+class MerchantItemsController < ApplicationController
+
+def show
+  @merchant = Merchant.find(params[:merchant_id])
+  @item = @merchant.items.find(params[:item_id])
```
Comment (ColinReinhart) [inline app/controllers/merchant_items_controller.rb:5]: Should we move lines 5 and 10 to a method?

Comment (tjhaines-cap) [reply app/controllers/merchant_items_controller.rb:5]: I think thats a good idea to make those lines into a model method.

**Review by tjhaines-cap — CHANGES_REQUESTED** [review]
Just a quick suggestion in the merchant items controller for a small change, everything else looks good.

**Comment by devAndrewK** [comment]
Made new branch Feat/Merchant Items Show Page AK

**Pull request closed without merging**